### PR TITLE
Creation of the badgen-cli

### PR DESCRIPTION
**Pull request by GMartigny**
Fix https://github.com/badgen/badgen/issues/33

### CODE REVIEW

**Comment by amio** [comment]
@GMartigny sorry for the late response, kinda busy these days. Will look at this soon, thanks!

File: readme.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+ * `--flat`, `-f`<br>
+    Boolean, use the flat style badge
+ * `--icon`, `-i`<br>
+    String, icon before the subject (icon name, image data URI or inferred from the subject)
+ * `--icon-width`, `-w`<br>
+    Number, width of the icon if not square
+
+## See also
+
+[Badgen](https://github.com/badgen/badgen), the module behind this CLI.
```
Comment (amio) [inline readme.md]:
```suggestion
[badgen](https://github.com/badgen/badgen), the module behind this CLI.
```

File: readme.md [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+
+## License
+
+[ISC](license)
```
Comment (amio) [inline readme.md]:
```suggestion
[MIT](/LICENSE)
```

File: cli.js [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+    Usage
+        $ badgen <options>
+
+    Options
+        --status, -s       (required) Status of the badge, right part
+        --subject, -j       Subject of the badge, left part
+        --color, -c         Color of the status
+        --flat, -f          Use the flat badge style
+        --icon, -i          Icon to use
+        --icon-width, -w    Width of the icon if not square
```
Comment (amio) [inline cli.js]: Some refinements

```
        -s, --status <text>            Status of the badge, right part (REQUIRED)
        -j, --subject <text>           Subject of the badge, left part
        -c, --color <RGB / ColorName>  Color of the status
        -f, --flat                     Use the flat badge style
        -i, --icon [<IconName>]        Icon to use
        -w, --icon-width <Number>      Width of the icon if not square
```

File: package.json [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+  "keywords": [
+    "cli",
+    "badge",
+    "svg"
+  ],
+  "bin": {
+    "badgen": "cli.js"
+  },
+  "scripts": {
+    "lint": "standart",
```
Comment (amio) [inline package.json]: I think we better use a broadly adopted linter, why not just `standard` 😉

Comment (GMartigny) [reply package.json]: OMG, that typo squatting!

File: cli.js (hunk 3 above)
Comment (GMartigny) [reply cli.js]: I just lowercased (required) because it felt too aggressive.

File: package.json (hunk 4 above)
Comment (amio) [reply package.json]: 😂

File: cli.js (hunk 3 above)
Comment (amio) [reply cli.js]: Yep, agreed

**Comment by tunnckoCore** [comment]
Is `process.exit()` defaults to code `0`?

**Comment by GMartigny** [comment]
@tunnckoCore Yes https://nodejs.org/api/process.html#process_process_exit_code

**Pull request merged**